src/veins/base/phyLayer/Frequencies.h

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (C) 2018 Dominik S. Buse <buse@ccs-labs.org>
+//
+// Documentation for these modules is at http://veins.car2x.org/
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+//
+
+#pragma once
+
+#include "veins/veins.h"
+#include <vector>
+
+using Frequency = double;
+using Powerlevel = double;
+using Frequencies = std::vector<Frequency>;
```

Comment:
How about moving this to a nested class of Spectrum? I really cannot see much use outside; right now the header file makes really strong statements about what a "frequency" or "frequencies" are supposed to mean within the whole framework.

Reply:
> right now the header file makes really strong statements about what a "frequency" or "frequencies" are supposed to mean within the whole framework.
That was the intention. Currently there is only one way to represent Frequencies and Powerlevels in the whole framework so rather have one place where they are defined.
However, I am fine with moving this to Spectrum for now.